website/content/docs/drivers/docker.mdx

```diff
@@ -1138,6 +1138,11 @@ many CPU shares are available to your process by reading `NOMAD_CPU_LIMIT`.
 Please keep the implications of CPU shares in mind when you load test workloads
 on Nomad.
 
+If resources [cores][cores] is set, the task is given an isolated reserved set of
+CPU cores to make use of. The total set of cores the task may run on is the private
+set combined with the variable set of unreserved cores. The private set of CPU cores
+is available to your process by reading `NOMAD_CPU_CORES`.
```

Comment:
Maybe link `NOMAD_CPU_CORES` to https://developer.hashicorp.com/nomad/docs/runtime/environment#job-related-variables?